content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -201,7 +201,7 @@ For a reference to old feature gates that are removed, please refer to
 | `SizeMemoryBackedVolumes` | `false` | Alpha | 1.20 | 1.21 |
 | `SizeMemoryBackedVolumes` | `true` | Beta | 1.22 | |
 | `StableLoadBalancerNodeGet` | `true` | Beta | 1.27 | |
-| `StatefulSetAutoDeletePVC` | `false` | Alpha | 1.22 | 1.26 |
+| `StatefulSetAutoDeletePVC` | `false` | Alpha | 1.23 | 1.26 |
```

Comment:
Checking the pkg/features/kube-feature.go, it says the gate was alpha since 1.22.
Also git log shows the feature was introduced on 2021-03-03, before 1.22 is out.
Can you confirm 1.23 is the right version?

Reply:
Looking at the implementation history in the enhancement and the subsequent PR of implementing _StatefulSetAutoDeletePVC_ changes, it appears this change is accurate.

**_Reference Links_**
- [KEP-1847: Auto delete PVCs created by StatefulSet
](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/1847-autoremove-statefulset-pvcs#implementation-history)
- https://github.com/kubernetes/kubernetes/pull/99728